### PR TITLE
Destkop file tweaks

### PR DESCRIPTION
**Pull request by keis**
Uzbl works better with gnome3 by grouping windows and the icon is no longer blurry

### CODE REVIEW

**Comment by keis** [comment]
I think you know distro packaging way better than I do @mathstuf, does this make sense?

**Comment by mathstuf** [comment]
I don't know that a `.desktop` file for `uzbl-core` makes sense. `uzbl-browser` (and `uzbl-tabbed`) are probably better candidates.

**Comment by keis** [comment]
Sorry. should have pasted the comment from the commit. But basically the idea is that the name of the desktop file should match the WM_CLASS we set of the window (which is uzbl-core). This makes at least gnome3 recoginsed those windows as belonging to uzbl and will correctly group them for switching within application windows (alt+`) and show the nice icon when doing alt+tab

**Comment by mathstuf** [comment]
+1

Ah, OK. There should probably additional `.desktop` files for the other instances as well too.

**Comment by keis** [comment]
Yeah. Guess it would make sense to have one for tabbed

File: uzbl-tabbed.desktop [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Uzbl Tabbed
+Comment=Uzbl - web interface tools which adhere to the unix philosophy.
+Terminal=false
+Type=Application
+Icon=uzbl.png
+Exec=uzbl-tabbed %u
```
Comment (mathstuf) [inline uzbl-tabbed.desktop]: Hmm. Isn't `%U` used for multi-url support?

Comment (keis) [inline uzbl-tabbed.desktop]: Ah, yes. you're right. looked at the firefox one but it looked the same with a simple %u

**Pull request merged**